nss-evening-cohort-24/social-justice-FE
# got edit meetups functionality working

## Description
Got edit meetups working 

## Related Issue

## Motivation and Context
Being able to edit the meetups after they've been created is a useful functionality.

## How Can This Be Tested?
Can be tested by navigating to the meetups page and clicking edit, changing info in the form, then submitting it and viewing the entry to confirm the changes.

## Screenshots (if appropriate):

## Types of changes

- [ x] New feature (non-breaking change which adds functionality)
